app/controllers/hyrax/admin/reports/works/work_activity_report_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Hyrax
+  class Admin::Reports::Works::WorkActivityReportController < ApplicationController
```

Comment:
Could these be combined?

Reply:
Right. So, the way the Menu Presenter currently works is based off of the controller_name. Which means the menu knows when to be collapsed/un-collapsed based on the active controller, https://github.com/samvera/hyrax/pull/2660/files#diff-6e8bbf07f89e869d65937ee64202ee7bR48

That's why currently each controller is individualized, and the routes are the way they are. But it probably makes sense to not allow the menu_presenter to dictate what routes and controllers we use, but to instead change the menu_presenter?